### ISSUE TITLE
Implement scraper for PA register

### ISSUE THREAD

**Issue by MthwRobinson** [civic-jabber/data-ingest#10]
The goal of this issue is to implement the code for downloading regulations from the [Pennsylvania register](https://www.pacodeandbulletin.gov/) and converting the output to XML. For now, you can store the resulting XML files in a local directory. The directory should use the following convention: `pa/{volume}/{number}/{regulation_number}.xml`.

The Pennsylvania register has the following disclaimer. Since we're not reselling the data or using it for profit, we should be good to go.

> No part of the information on this site may be reused for profit or sold for profit.

**Comment by kvnkho** [comment]
Does a parser really need to be to handle XML or can we do it in JSON then convert?

**Comment by MthwRobinson** [comment]
The reason for the XML is:

1. A lot of the regulations (like [this one](http://register.dls.virginia.gov/details.aspx?id=8119)) have strike throughs and other markdown formatting that would be useful to maintain.
1. I'd like to enforce use of a [common schema](https://github.com/civic-jabber/data-ingest/blob/master/data_ingest/schemas/regulation.xsd), which is a little easier in XML.
1. I find XML to be more human readable;

That said, I am planning on implementing a helper function that converts a `dict` to XML, so if you can get all the data into a dictionary, you an just call that.

**Comment by kvnkho** [comment]
Got it that makes sense. Yeah, I was wondering about the VA code was going to be re-written or just converted. If converted, that helper function would be useful across the registers.

**Comment by MthwRobinson** [comment]
Yeah, I'm planning to make the changes to the VA code today. Can link that in here so people can use it as an example

**Comment by MthwRobinson** [comment]
The `Regulation` data model now has a `to_xml` method now that you can call. If you can turn your output into a `Regulation` object, you should be good to go. Here's an example of what it looks like for Virginia.

https://github.com/civic-jabber/data-ingest/blob/e72c77581f83c09c7fe98aa01e0e821c24514641/data_ingest/regs/va.py#L59-L61

https://github.com/civic-jabber/data-ingest/blob/e72c77581f83c09c7fe98aa01e0e821c24514641/data_ingest/regs/va.py#L113-L154
